Tools/Scripts/test262-helpers/README.md

```diff
@@ -1,16 +1,31 @@
-Install [Perlbrew](https://perlbrew.pl/)
+To install Perl 5.8.8, use [Perlbrew](https://perlbrew.pl/)
+
+Mac:
 
 `\curl -L https://install.perlbrew.pl | bash`
 
+Linux (Debian derivative):
+
+```
+sudo apt-get install perlbrew
+perlbrew init
+```
+
 Append the following piece of code to the end of your ~/.bash_profile and start a
 new shell, perlbrew should be up and fully functional from there:
 
 `source ~/perl5/perlbrew/etc/bashrc`
 
 Install Perl 5.8.8 through Perlbrew
 
+Mac:
+
 `perlbrew install perl-5.8.8`
 
+Linux:
+
+`perlbrew --notest install perl-5.8.8   # Perl 5.8.8 has some known compilation errors`
```

Comment:
the errors on 5.8.8 are true for any system, they are not really compilation errors, but the tests are known failures that won't be fixed for 5.8.8.